### PR TITLE
FR-welcome post

### PR DESCRIPTION
**Pull request by amsichani**
blog post welcoming Marie & Sofia and announcing the FR initiative

### CODE REVIEW

**Comment by mdlincoln** [comment]
Thanks for pushing this! The error with the image link is caused by the fact that the `_promo` folder technically doesn't get published to the web. Give me a few minutes and I can modify that.

File: _posts/2018-07-13-FR-team.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Programming Historian French initiative 
+authors:
+- Anna-Maria Sichani
+layout: post
+categories: posts
+---
+
+<p><figure><img src="/avatars/Marie-Puren.png" alt=""/><figcaption>
```
Comment (mdlincoln) [inline _posts/2018-07-13-FR-team.md]: Fix the nesting of these tags please. Both `` elements can sit inside one `<figure>` element.

File: _posts/2018-07-13-FR-team.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+<figure><img src="/avatars/Sofia-Papastamkou.png" alt=""/><figcaption>
+Marie Puren and Sofia Papastamkou have joined the Programming Historian</figcaption></figure>
+</p>
+
+We are pleased to announce that Marie Puren and Sofia Papastamkou have joined the Programming Historian project team. By welcoming Marie and Sofia as members of the editorial board we are also introducing the French-speaking initiative of the Programming Historian. 
+
+Marie Puren Ph.D., is a junior researcher in Digital Humanities at the French National Institute for computer science and applied mathematics (Inria) in Paris, member of the ALMAnaCH team (INRIA – Ecole Pratique des Hautes Etudes). As collaborator to the [PARTHENOS] (http://www.parthenos-project.eu/) H2020 project, she focuses her research on the development of standards for data management and research tools in Arts and Humanities. Marie also contributes to the [IPERION] (http://www.iperionch.eu/) H2020 project.
```
Comment (mdlincoln) [inline _posts/2018-07-13-FR-team.md]: Do not put a space in between the `[]` part of a markdown URL and the `()` part. See https://programminghistorian.org/en/lessons/getting-started-with-markdown#links

**Review by mdlincoln — CHANGES_REQUESTED** [review]

**Comment by amsichani** [comment]
fab - thanks @mdlincoln (and sorry for the wrong branch name)!

File: _posts/2018-07-13-FR-team.md (hunk 2 above)
Comment (amsichani) [reply _posts/2018-07-13-FR-team.md]: done!

**Review by mdlincoln — APPROVED** [review]

**Comment by mdlincoln** [comment]
This looks good to me so far - the only error is that ADHO 404 in the DH2018 post due to ongoing problems in their servers. Are we happy publishing @amsichani ? If so I'll override the errors and merge it.

**Comment by amsichani** [comment]
I think we are good to go! thanks @mdlincoln !

**Comment by amsichani** [comment]
I just saw it : we need to change the blog title to "French-speaking" - how do I proceed @mdlincoln ? new branch etc?

**Pull request merged**